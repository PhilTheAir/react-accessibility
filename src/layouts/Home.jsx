import React from 'react';
import PageContainer from '../containers/PageContainer';

const Home = props => (
  <PageContainer />
);

export default Home;