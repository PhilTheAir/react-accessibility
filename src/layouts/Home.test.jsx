import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home', () => {
  it('renders the home page structure', () => {
    const component = <Home />;
    const wrapper = shallow(component).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
