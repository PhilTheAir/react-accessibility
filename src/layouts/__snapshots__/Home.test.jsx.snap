// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the home page structure 1`] = `
<Styled(Flex)
  alignItems="center"
  justify="center"
>
  <LinkIcon
    icon={<IoSocialAndroid />}
    key="Android"
    link="https://www.android.com"
    title="Android"
  />
  <LinkIcon
    icon={<IoSocialNodejs />}
    key="Node.js"
    link="https://nodejs.org/en/"
    title="Node.js"
  />
  <LinkIcon
    icon={<IoSocialGithub />}
    key="GitHub"
    link="https://github.com"
    title="GitHub"
  />
</Styled(Flex)>
`;
